protected/humhub/widgets/CoreJsConfig.php

```diff
@@ -270,8 +270,12 @@ public function run()
                         'success.unarchive' => Yii::t('ContentModule.base', 'The content has been unarchived.'),
                         'success.delete' => Yii::t('ContentModule.base', 'The content has been deleted.'),
                         'info.editCancel' => Yii::t('ContentModule.base', 'Your last edit state has been saved!'),
+                        'info.updatesAvailable' => Yii::t('ContentModule.base', 'New Updates Available!')
```

Comment:
Any reason why we need this twice?

Reply:
Just forgot to remove it, I've fixed it already.